packages/ui/src/icons/relations-icon.tsx

```diff
@@ -9,7 +9,7 @@ export const RelationsIcon: React.FC<ISvgIcons> = ({ className = "text-current",
     stroke="currentColor"
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
-    strokeWidth="1.5"
+    // strokeWidth="1.5"
```

Comment:
why did we reduce the stroke width? It looks noticeably different to other icons.

Reply:
Resolved